From 079e914b171bf854a2f5e548e75367201791c3e3 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Sat, 8 Sep 2018 14:42:49 -0700
Subject: [PATCH 24/26] ro.crypto.{allow_encrypt_override filenames_mode}
 vendor-init-settable.

Bug: 114017832
Test: m selinux_policy
Change-Id: I1dcb09c76b3e49888d278a154d79add6c6a6c977
Merged-In: I1dcb09c76b3e49888d278a154d79add6c6a6c977
(cherry picked from commit fe72cb70d75479d726ae090ea871a7de943bad41)
---
 public/property_contexts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/public/property_contexts b/public/property_contexts
index f252b4a3..8e53eef8 100644
--- a/public/property_contexts
+++ b/public/property_contexts
@@ -98,7 +98,9 @@ ro.config.notification_sound u:object_r:exported2_config_prop:s0 exact string
 ro.config.ringtone u:object_r:exported2_config_prop:s0 exact string
 ro.control_privapp_permissions u:object_r:exported3_default_prop:s0 exact string
 ro.cp_system_other_odex u:object_r:exported3_default_prop:s0 exact int
+ro.crypto.allow_encrypt_override u:object_r:exported2_vold_prop:s0 exact bool
 ro.crypto.scrypt_params u:object_r:exported2_vold_prop:s0 exact string
+ro.crypto.volume.filenames_mode u:object_r:exported2_vold_prop:s0 exact string
 ro.dalvik.vm.native.bridge u:object_r:exported_dalvik_prop:s0 exact string
 ro.enable_boot_charger_mode u:object_r:exported3_default_prop:s0 exact bool
 ro.gfx.driver.0 u:object_r:exported3_default_prop:s0 exact string
-- 
2.17.1

